frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -31,6 +31,7 @@ import VersionButton from './VersionButton';
 export const drawerWidth = 240;
 export const mobileDrawerWidth = 320;
 export const drawerWidthClosed = 64;
+export const topBarHeight = 64;
```

Comment:
The problem is that the sidebar is set from the top of the screen, without having the toolbar before it.
A more future-proof way of fixing this issue is to see what the normal case is doing: Adding a Box with the toolbar style mixin (look for the use of the `toolbar` class).